### ISSUE TITLE
Remote server

### ISSUE THREAD

**Issue by AnaBVA** [maxsonBraunLab/CITEViz#2]
Hi, 
Nice app. 
I'm working on RStudio Server and the seurat obj is not in my local machine. Can I use a parameter to run the Shiny app with the object previously selected when I execute `CITEViz::run_app(options = list(rds_input_file=here::here("seurat.RDS")))`? 
Thanks

**Comment by gartician** [comment]
Hi @AnaBVA,

Thank you for your interest in the app, you attended Bioconductor 2022 online right? The app currently supports RDS files in your local computer because of the default behavior of the "Input" button in R Shiny, but I agree it would be convenient to feed in the RDS through the command line.

**Comment by AnaBVA** [comment]
Yes!! I was in BioC2022 :D and wanted to try CITEViz in my own data.
Nice app, I definitely can see the value of it.
